utilidades/etl_date_utils.py

```
@@ -1,202 +1,19 @@
-""" Contains deprecated TimeUtils class/methods and
-current implementation of uitlity datetime conversion methods 
-used in ETL pipeline (DateUtilsETL class)"""
 
-__all__ = ['TimeUtils', "DateUtilsETL"] #Export the class
+
+__all__ = ['DateUtilsETL'] #Export the class
 
 import pandas as pd
 import pytz
 from datetime import datetime
 from typing import Dict, Union, Tuple
```

Ruff (0.12.2)

`typing.Union` imported but unused

Remove unused import

(F401)